blog/BCHN-lead-maintainer-report-2020-06-15.md

```diff
@@ -0,0 +1,350 @@
+
+I'll be focusing a large part of my efforts in the next 2-3 weeks on the rolling checkpoints specification and validation. I'm behind on the specification parts, but think I can make up sufficient ground.
+
+On the website improvement work package, [Corentin Mercier](https://merc1er.com/) (@merc1er) has moved the project forward significantly by setting up translations via Crowdin and building a Newsroom page to which we can add content.
+
+<figure class="text-center">
+    <img src="/static/img/newsroom/2020-06-15/Fr4hoBC9t5dYGodpfRAizdoSxBQTrGYkAjM4OGmA.png" class="rounded img-fluid" alt="" title="">
+</figure>
+
+He is current working on improving the user experience of our site (front page needs an overhaul) and [progressing Flipstarter deployment capability](https://gitlab.com/flipstarter/backend/-/issues/27) via a project jointly funded by BCHN and the Flipstarter team. This will allow BCHN to easily deploy Flipstarter technology to raise funds in future for further development.
+
+## [Additional improvements](#additional-improvements)
+
+### [New API: `getblocktemplatelight` + `submitblocklight`](#new-api-getblocktemplatelight-submitblocklight)
+
+This is a major new RPC interface developed by Calin Culianu in conjunction with WaYi / easy2mine pool.
+
+It offers a significant speed improvement to pools over the legacy `getblocktemplate`/`submitblock` method. BCHN published an earlier [Technical Bulletin](https://read.cash/@bitcoincashnode/bchn-technical-bulletin-2020-05-04-51b0305f) about the performance improvements. The new feature comprised significant amounts of new code and is a great benefit for mining pools.
+
+Since any defect in such critical mining code could result in pools losing money due to invalid blocks or downtime, it was decided to spend some extra funds on review of this code. BCHN paid out two bounties to project contributors for review of this development (these were announced in our Slack developer channel). In return the project obtained a high quality of code reviews and could confidently release this new feature in v0.21.2.
+
+## [Performance optimizations to JSON library and RPC interface](#performance-optimizations-to-json-library-and-rpc-interface)
+
+BCHN developers continue to improve the performance of JSON RPC, chiefly through optimization of the Univalue library, but not limited to that.
+
+This work is driven by BCHN developers @BigBlockIfTrue and Calin Culianu.
+
+Significant improvements have been achieved since BCHN project start, and since v0.21.2 release. More of these will be available in our next minor release v0.21.3.
+
+Work is progressing toward catching up with the performance of [rapidjson](https://rapidjson.org/), the leading JSON library in the field. Currently UniValue's JSON is slower by a factor of 2. This isn't actually that much difference anymore and we hope to release more benchmark results in the near future to showcase relevant BCHN improvements.
+

```

Comment:
There are two paragraphs from the original which are completely missing here!

Reply:
Thanks @ftrader for finding this... i completely missed those, added them back